modules/monitoring-examining-metrics-as-a-developer.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="examining-metrics-as-a-developer_{context}"]
+= Examining metrics as a developer
+
+You might enable user workload monitoring for an application or service in a project. In that case, a developer of that project can examine the exposed custom metrics using the Developer Perspective in the Web console.
```

Comment:
```suggestion
As a developer one can enable user workload monitoring for an application or service in a project. As an admin you use the same feature to enable monitoring for infrastructure workloads. In that case, a developer or admin of that project can examine the exposed metrics using the Developer Perspective in the Web console.
```

Reply:
Committed, but this sentence seems out of place:
"As an admin you use the same feature to enable monitoring for infrastructure workloads."
Why are we putting the information pertaining to _infrastructure_ workloads when this "Examining metrics as a developer" module and the outer section "User workload monitoring" pertains only to _user_ workloads? Plus, does the administrator actually need to enable monitoring for infrastructure workloads?

Reply:
Never mind the previous comment. Suggestion implemented.